torch/ao/quantization/fx/quantization_patterns.py

```diff
@@ -1033,10 +1033,8 @@ def convert(self,
                         bias = load_arg(quantized=torch.float)(self.linear_node.args[2])
                         other_args = other_args[1:]  # remove the bias argument
                     else:
-                        assert 'bias' in kwargs, \
-                            'expect bias provided as a keyword argument when it is not a positional argument'
-                        bias = kwargs['bias']
-                        kwargs.pop('bias')
+                        kwargs.pop('bias', None)
```

Comment:
can you add a comment in the code here? (maybe copying the comment in the github) I thought we will either have positional or keyword bias arg

Reply:
>I thought we will either have positional or keyword bias arg

AFAIK fx traces the arguments as they were passed to `__torch_function__` which for C++ functions will match how the user called it. So as long as the signature allows it, any argument could be either positional or keyword.